### ISSUE TITLE
False positive closure_spacing with only comment inside

### ISSUE THREAD

**Issue by Igor-Palaguta** [realm/SwiftLint#4565]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

closure_spacing is fired when closure contains only comment

##### Complete output when running SwiftLint, including the stack trace and command used

```bash
$ swiftlint lint
```

`warning: Closure Spacing Violation: Closure expressions should have a single space inside each brace. (closure_spacing)`

### Environment

* SwiftLint version (run `swiftlint version` to be sure)? 0.50.0
* Installation method used (Homebrew, CocoaPods, building from source, etc)? Cocoapods
* Paste your configuration file:
```yml
opt_in_rules:
  - closure_spacing
```

* Are you using [nested configurations](https://github.com/realm/SwiftLint#nested-configurations)?
  no
* Which Xcode version are you using (check `xcodebuild -version`)? 
Xcode 14.0
Build version 14A309
* Do you have a sample that shows the issue? Run `echo "let test1 = func1(arg: { /* do nothing */ })" | swiftlint lint --no-cache --use-stdin --enable-all-rules`
  to quickly test if your example is really demonstrating the issue. If your example is more
  complex, you can use `swiftlint lint --path [file here] --no-cache --enable-all-rules`.

```swift
// This triggers a violation:
let test1 = func1(arg: { /* do nothing */ })
let test2 = func1 { /* do nothing */ }
```

**Comment by jpsim** [comment]
Fixed in https://github.com/realm/SwiftLint/pull/4584
